draft-rokui-ccamp-actn-wdm-pluggable-modelling.md

```diff
@@ -369,21 +410,23 @@ The host side of the coherent pluggable is further divided into two functional b
 
 The characteristics of the Electrical Channels are:
 
-* The host interface lanes of the device forming the physical interface to the host platform data path device(s)
+Note that purpose of this section is to clarify the coherent pluggable. This purpose of this draft is not to define the data model of the electrical channels.
```

Comment:
for me the wording is still not clear. Please follow my suggestion above.

Reply:
Can you suggest an alternative text. I will add it

Reply:
I would reword a bit : " Note that the purpose of this section is to clarify the role of electrical channel in the coherent pluggable"
and "The purpose of this draft is not ...."

Reply:
Done